ADRs/0004-separate-consensus-and-p2p-keys.md

```diff
@@ -0,0 +1,52 @@
+# Separate Consensus and P2P Private Keys
+
+* Status: in review
+* Deciders: Pocket Network Team
+* Date: 2023-04-17
+
+Technical Story: [Consolidate and Refactor Node Identity](https://github.com/pokt-network/pocket/issues/348)
+
+## Context and Problem Statement
+
+In the context of maintaining a secure and robust network, we need to decide whether to use separate private keys for P2P and consensus modules, or use a single private key for both purposes.
+Combining the keys could simplify the identity management, but may also increase security risks and impact system quality.
+
+## Decision Drivers
+
+* Security: Minimizing the risks associated with key compromise
```

Comment:
NIT: bold the reasons

Reply:
Can you elaborate? Are you referring to the `Minimizing the risks...` part?

Reply:
@Olshansk (bump)